dfastmi/__main__.py

```diff
@@ -113,7 +113,7 @@ def parse_arguments() -> Tuple[str, str, Optional[str], str, bool]:
         "--rivers",
         help="name of rivers configuration file ('Dutch_rivers.ini' is default)",
     )
-    parser.set_defaults(rivers="Dutch_rivers.ini")
+    parser.set_defaults(rivers="unspecified")
```

Comment:
Is there a specific reason to put it to unspecified, since lateron the riverfile will be put to a file based on the runmode.
Wouldn't it be better to use that if here instead of using the "unspecified"?
e.g.:

`if runmode == "CLI":
            parser.set_defaults(rivers="Dutch_rivers_v1.ini")"
        else:
            parser.set_defaults(rivers="Dutch_rivers.ini")`

Reply:
The reason is that the `runmode` is not yet known ... that is one of the arguments to be parsed. Basically, we check whether the `rivers` argument has been specified or not. If not, then we use the default value which depends on the `runmode` (_v1 for the backward compatible CLI mode, and normal version ... maybe we should rebrand it to _v2 for the BATCH and GUI mode)

Reply:
Note: I've now renamed the new Dutch_rivers.ini file to Dutch_rivers_v2.ini to make it more clear that this is a new file.